src/screens/SUE/SueListScreen.tsx

```diff
@@ -81,7 +107,23 @@ export const SueListScreen = ({ navigation, route }: Props) => {
   const listItems = useMemo(() => {
     if (!data?.length) return [];
 
-    return parseListItemsFromQuery(query, data, undefined, { appDesignSystem }).reverse();
+    let parsedListItem = parseListItemsFromQuery(query, data, undefined, {
+      appDesignSystem
+    }).reverse();
+
+    if (queryVariables.sortBy) {
+      const { sortBy } = queryVariables;
+
+      parsedListItem = parsedListItem.sort((a, b) => {
+        if (sortBy === SORT_BY.REQUESTED_DATE_TIME || sortBy === SORT_BY.UPDATED_DATE_TIME) {
+          return new Date(b[sortBy]) - new Date(a[sortBy]);
+        }
+
+        return a[sortBy].localeCompare(b[sortBy]);
+      });
```

Comment:
does this logic reflect the same results as in the old melde.möwe app filters?

Reply:
Except for Status, all sorting is the same as Melde.Möwe. I think they have a specific algorithm for Status. At the moment, when we sort by status, we sort according to the alphabetical order of the statuses.
In our app, status sorting:
```
[
  'Geschlossen',
  'In Bearbeitung',
  'Offen',
  'Unbearbeitet',
  'Ungültig',
  'Wartend auf Dritte',
  'Wartend auf Meldenden',
];
```
Status sorting in the Melde.Möwe app (In my mind):
```
[
  'Offen',
  'Unbearbeitet',
  'In Bearbeitung',
  'Wartend auf Dritte',
  'Wartend auf Meldenden',
  'Geschlossen',
  'Ungültig'
];
```
I need to update the sorting in this way.

Reply:
Currently, when we sort by status, it is sorted as in the Melde.Möwe app.